aspnetcore/fundamentals/hosting.md

```diff
@@ -250,33 +559,360 @@ using (host)
 }
 ```
 
-The URL formats that are valid here depend on the server you're using. For more information, see [Server URLs](#server-urls) earlier in this article.
+You can initialize and start a new host using the pre-configured defaults of `CreateDefaultBuilder` using a static convenience method. These methods start the server without console output and with [WaitForShutdown](/dotnet/api/microsoft.aspnetcore.hosting.webhostextensions.waitforshutdown) wait for a break (Ctrl-C/SIGINT or SIGTERM):
 
-> [!NOTE]
-> The `UseConfiguration` extension method isn't currently capable of parsing a configuration section returned by `GetSection` (for example, `.UseConfiguration(Configuration.GetSection("section"))`. The `GetSection` method filters the configuration keys to the section requested but leaves the section name on the keys (for example, `section:urls`, `section:environment`). The `UseConfiguration` method expects the keys to match the `WebHostBuilder` keys (for example, `urls`, `environment`). The presence of the section name on the keys prevents the section's values from configuring the host. This issue will be addressed in an upcoming release. For more information and workarounds, see [Passing configuration section into WebHostBuilder.UseConfiguration uses full keys](https://github.com/aspnet/Hosting/issues/839).
+**Start(RequestDelegate app)**
+
+Start with a `RequestDelegate`:
+
+```csharp
+using (var host = WebHost.Start(app => app.Response.WriteAsync("Hello, World!")))
+{
+    host.WaitForShutdown();
```

Comment:
It doesn't quite work for this. `WaitForShutdown` blocks until a shut down signal is given so the message `Press any key to exit...` does not get printed until the host is being shutdown. Also, `Console.ReadKey` is no longer needed. What you want here is
```
using (var host = WebHost.Start(app => app.Response.WriteAsync("Hello, World!")))
{
    Console.WriteLine("Press any key to exit...");
    host.WaitForShutdown();
}
```
Note there are a few more instance of this.

Reply:
>  the message Press any key to exit... does not get printed until the host is being shutdown

Yes, but that's the point of doing that ... to keep the console window open. I guess it would have made more sense if I had done it this way ...

```csharp
 using (var host = WebHost.Start(app => app.Response.WriteAsync("Hello, World!")))
 {
    Console.WriteLine("Use Ctrl-C to shutdown the host...");
    host.WaitForShutdown();

    Console.WriteLine("Press any key to exit...");
    Console.ReadKey();
 }
 ```

Slight misunderstanding on what I was trying to do here and on the [sample update PR](https://github.com/aspnet/MetaPackages/pull/225), when @Tratcher said, "it prevents the Console.WriteLine." I'm trying to keep the console window open (and demo that additional code can execute) after the host shuts down.

However, it's good to just state in advance what the user should do ("Ctrl-C") and what will happen ("shutdown the host"), so I'll go with that on the next commit.